graphql/root.js

```diff
@@ -1,3 +1,29 @@
-const root = { mostWins: () => "Hello world!", mostLosses: () => "Bye world!" };
+const root = { 
+  mostWins: ({numUsers}) => topOfLeaderboard(numUsers),
+  mostLosses: ({numUsers}) => bottomOfLeaderboard(numUsers),
+  highestWinStreaks: ({numUsers}) => topWinStreaks(numUsers),
+  highestLoseStreaks: ({numUsers}) => topLoseStreaks(numUsers),
+  winners: ({userHandles}) => winners(userHandles)
+};
+
+function topOfLeaderboard(numUsers) {
+  return ["@ivan.chan", "@brian.zhang", "@nashua.luk"];
+}
+
+function bottomOfLeaderboard(numUsers) {
+  return ["@dummy", "@data"];
+}
+
+function topWinStreaks(numUsers) {
```

Comment:
We should make a default value instead of taking in arguments. For ex: always return top 10 winstreaks

Reply:
depends on how we plan to implement it, out of this PRs scope